language-design.md

```diff
@@ -38,6 +71,10 @@ How do the themes of _Growing a Language_ relate to the lab we did this week?
 
 **Response**
 
+An overarching theme of Growing a Language was of limited development time.  Because developers cannot predict all the requirements that users of their language might have, some development choices must be left to a later time [Steele, 1998, p 9].  Trying to accomplish everything with a language out the gate would take so long that your language would likely become obsolete before it’s complete.  In our lab we were similarly constrained by time, as we had only thirty minutes to design our approach and implement whatever we could.  In order to achieve anything in such a short amount of time, an approach that we discussed was starting with small pieces and building up (the lab guidelines urged us to “use a limited set of tools”).  This relates to general language design where, simply put, one must “add to the small language to make a language that is more large” [Steele, 1998, p 4]. 
```

Comment:
Hadn't even considered this, good point. And I guess we were intentionally very limited on time by Prof. Ben to get us in that mode of thinking about the things that are absolutely most important.